Description: Fix paths to frobby and 4ti2 in frobgrob script.
Author: Doug Torrance <dtorrance@piedmont.edu>
Forwarded: not-needed
Last-Update: 2021-11-15

--- a/frobgrob
+++ b/frobgrob
@@ -2,16 +2,14 @@
 file="$1" # the input file
 shift # important as any remaining options are passed on to Frobby
 
-fti2=4ti2/4ti2gmp # where to find 4ti2
-
 # make input file $file.out.mat from $file in the format that 4ti2 expects
 echo -n "1 " > $file.out.mat
 wc $file | sed "s/ *[0-9]* *\([0-9]*\) *.*/\1/" >> $file.out.mat
 cat $file >> $file.out.mat
 
 # run 4ti2 to compute Grobner basis
-$fti2 zbasis $file.out > /dev/null # make $file.out.lat (not required)
-$fti2 groebner $file.out > /dev/null # make $file.out.gro
+4ti2-zbasis $file.out > /dev/null # make $file.out.lat (not required)
+4ti2-groebner $file.out > /dev/null # make $file.out.gro
 
 # run frobby to compute Frobenius number from Grobner basis
-cat $file.out.gro $file | bin/frobby optimize -chopFirstAndSubtract -maxStandard $*
+cat $file.out.gro $file | frobby optimize -chopFirstAndSubtract -maxStandard $*
